templates/test/ci/prow-aks/patches/aks-pool0.yaml

```diff
@@ -7,7 +7,6 @@ spec:
   osDiskType: "Managed"
   osDiskSizeGB: 30
   enableNodePublicIP: false
-  enableUltraSSD: true
```

Comment:
Looks good. Did you check if ultra SSDs are available in the new regions?

Reply:
They evidently are not in switzerlandnorth, but this removes all usage of UltraSSD in our e2e tests so we shouldn't be hitting that kind of error anymore.